qt_ui/widgets/QBudgetBox.py

```diff
@@ -14,18 +14,24 @@ def __init__(self, game):
         super(QBudgetBox, self).__init__("Budget")
 
         self.game = game
-        self.money_icon = QLabel()
-        self.money_icon.setPixmap(CONST.ICONS["Money"])
-        self.money_amount = QLabel()
+        # self.money_icon = QLabel()
+        # self.money_icon.setPixmap(CONST.ICONS["Money"])
+        # self.money_amount = QLabel()
```

Comment:
I'm fairly certain this was updated from many locations (when buying/selling units, for example), and that those cases are now broken since they weren't updated to match this.

Reply:
It appears the setGame method in the QBudgetBox is used to update the money_amount label, which itself calls setBudget. I've tested buying and selling things from an airport and disbanding and hiring site locations and they all update the text on the button properly.